src/main/java/com/example/webapp/Model/Department.java

```diff
@@ -0,0 +1,71 @@
+package com.example.webapp.Model;
+
+import com.example.webapp.dto.DepartmentDTO;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import java.util.Set;
+
+@Entity
+@Table
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
I think NoArgsConstructor AllArgsConstructor are already in  Data ?

Reply:
They are not